Update virtual-memory.md (#1788)
redis/redis-doc: topics/virtual-memory.md

Virtual Memory
===

> Redis Virtual Memory is a feature that was available since Redis 2.0 till
> Redis 2.4 and is now deprecated.

Reviewer: ```suggestion
Redis Virtual Memory is a feature that was available since Redis 2.0.
It has been deprecated as of Redis 2.4.
```
